pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go

```diff
@@ -66,6 +72,12 @@ func NewCSIDriverControllerServiceController(
 	optionalInformers = append(optionalInformers, configInformer.Config().V1().Infrastructures().Informer())
 	var optionalManifestHooks []dc.ManifestHookFunc
 	optionalManifestHooks = append(optionalManifestHooks, WithPlaceholdersHook(configInformer))
+	optionalManifestHooks = append(optionalManifestHooks,
+		WithLeaderElectionReplacerHook(
+			leaderelection.LeaderElectionDefaulting(
```

Comment:
ugly indention, hard to read

Reply:
if you moved it into a local variable `libraryGoLeaderElectionsDefaults` is would be trivial to read. Now I had to check two functions to understand.

Reply:
Reworked to a variable